proto/cosmos/offchain/v1alpha1/offchain.proto

```diff
@@ -0,0 +1,15 @@
+syntax="proto3";
+
+package cosmos.offchain.v1alpha1;
+
+option go_package="github.com/cosmos/cosmos-sdk/x/offchain";
+
+import "gogoproto/gogo.proto";
+
+// MsgSignData defines an arbitrary, general-purpose, off-chain message
+message MsgSignData {
+    // signer is the sdk.AccAddress of the message signer
+    bytes signer = 1 [(gogoproto.jsontag) = "signer", (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    // data represents the raw bytes of the content that is signed (text, json, etc)
+    bytes data = 2 [(gogoproto.jsontag) = "data"];
```

Comment:
i believe omitting the `jsontag` option is okay here, as that would be the default output anyways. (might be good to double check: remove this option, re-generated the .pb.go files, make sure the generated files still have a `json:` tag in Go structs)

Reply:
changes addressed